docker-compose.yaml

```diff
@@ -0,0 +1,31 @@
+services:
+  db:
+    image: postgres:13.18-alpine3.20
+    restart: always
+    env_file:
+      - .env
+    ports:
+      - "5432:5432"
+    volumes:
+      - my_db:$PGDATA

```

Comment:
Ensure that the environment variable `PGDATA` is correctly set in the `.env` file or elsewhere in your environment. This variable should point to the directory where PostgreSQL stores data.